Numbers.java

```diff
@@ -0,0 +1,109 @@
+package Lesson_12;
+
+import java.math.BigInteger;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Scanner;
+
+public class Numbers {
+    ArrayList<Integer> numbersList = new ArrayList<>();
+    char choice;
+    boolean isPrime=true;
+
+    public Numbers (){
+        this.numbersList=numbersList;
+        this.choice=choice;
+        this.isPrime=isPrime;
+    }
+
+
+    public ArrayList addNumbersToList(char inputChoice) {
+        Scanner reader = new Scanner(System.in);
+        Integer addNumber = 0;
+        if (inputChoice == 'y') {
+            do {
+                System.out.println("Type any number:");
+                addNumber = reader.nextInt();
+                this.numbersList.add(addNumber);
+                System.out.println("Add another number? y/n");
+                this.choice = reader.next().toLowerCase().charAt(0);
+            } while (this.choice == 'y');
+        }else System.out.println("Ok. Thank you.");
+        return this.numbersList;
+    }
+
+    public void printList (){
+        System.out.println(this.numbersList);
```

Comment:
It would be good to print output of the list by iterating over the list and printing each element specifically